EIPS/eip-3675.md

```diff
@@ -128,7 +128,12 @@ Changes to the block tree store that are related to the above actions **MUST** b
 
 #### EIP-2124 fork identifier
 
-For the purposes of the [EIP-2124 fork identifier](https://eips.ethereum.org/EIPS/eip-2124), nodes implementing this EIP **SHOULD** keep values of `FORK_NEXT` and `FORK_HASH` unaffected, i.e. `FORK_NEXT` is set to `0` and `FORK_HASH` retains the same value.
+For the purposes of the [EIP-2124 fork identifier](./eip-2124.md), nodes implementing this EIP **MUST** keep values of `FORK_NEXT` and `FORK_HASH` unaffected, i.e. `FORK_NEXT` is set to `0` and `FORK_HASH` retains the same value.
+
+Once the transition has happened, `TRANSITION_POS_BLOCK` will have some block number `N`.
+This block number cannot be known ahead of time given the dynamic nature of the transition trigger condition.
+Starting with `TRANSITION_POS_BLOCK`, nodes **MUST** update `FORK_HASH` by appending `uint64(N)` to the sequence of hashes used to calculate `FORK_HASH` as per EIP-2124.
```

Comment:
If I understand EIP-2124 correctly, a local node will have to reject the connection with the remote node if the `FORK_HASH` of the remote node can't be completed with the locally known `FORK_NEXT`. This will be the case for the local node that is syncing and haven't reached the `TRANSITION_POS_BLOCK` yet while the transition has already happened in the network and the `TRANSITION_POS_BLOCK` is known to the remote peer, thus, the remote peer has updated its `FORK_HASH` value with the transition block.